lang/cs/Org.Apache.REEF.Client/Common/DriverFolderPreparationHelper.cs

```diff
@@ -116,6 +119,18 @@ internal void CreateDefaultFolderStructure(string driverFolderPath)
         /// </summary>
         private void AddAssemblies()
         {
+            Assembly assembly = typeof(DriverFolderPreparationHelper).Assembly;
+            String[] names = assembly.GetManifestResourceNames();
+            ResourceSet resourceSet = new ResourceSet(assembly.GetManifestResourceStream(names[0]));
+
+            var clientJarBytes = resourceSet.GetObject("reef_bridge_client") as byte[];
```

Comment:
These constants are potentially brittle, right? As far as I understand, they are used in code twice, and once in a build file. Can we at least turn them into constants in the code, and add the location of the code constants to the build file as a comment?